common/test/acceptance/tests/lms/test_progress_page.py

```diff
@@ -142,6 +142,7 @@ def _change_subsection_structure(self):
         subsection = self.course_outline.section(self.SECTION_NAME).subsection(self.SUBSECTION_NAME)
         subsection.expand_subsection()
         subsection.add_unit()
+        subsection.wait_for_ajax()
```

Comment:
`wait_for_ajax` is sometimes a bit too much solution for a problem (like using a chainsaw to cut a piece of paper), is there anything more targeted we could wait for? Some element that exists after the important parts of `add_unit` are finished?